src/js/themes/dark.js

```diff
@@ -18,7 +18,7 @@ const lightColors = ['#333333', '#444444', '#555555', '#666666', '#777777', '#99
 const darkColors = ['#F6F6F6', '#EEEEEE', '#DDDDDD', '#CCCCCC', '#BBBBBB', '#AAAAAA'];
 const backgroundColor = '#111111';
 const textColor = '#eeeeee';
-const borderColor = 'rgba(255, 255, 255, 0.33)';
+const borderColor = 'rgba(255, 255, 255, 0.50)';
```

Comment:
why was this changed?

Reply:
The brand color is pretty light, so the border is barely visible:
![screenshot701](https://user-images.githubusercontent.com/6075606/39080173-7e929100-44f7-11e8-8964-2a1e33aac20e.jpg)